views/post.py

```diff
@@ -132,3 +132,37 @@ def get_all_posts():
             posts.append(post)
 
     return json.dumps(posts)
+
+
+def create_post(user_id, category_id, title, content, image_url=None):
+    """
+    Creates a new post in the database.
+
+    Returns:
+        bool: True if the post was created successfully, False otherwise.
+    """
+    publication_date = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
In the starter code that was provided for us in `users.py`, they just use `datetime.now()` for the date. So we can remove the `.strftime("%Y-%m-%d %H:%M:%S")` conversion, to keep things consistent.